torch/csrc/autograd/saved_variable.h

```diff
@@ -29,7 +29,9 @@ class TORCH_API SavedVariable {
       const std::optional<Variable>& variable,
       bool is_output,
       bool is_inplace_on_view = false);
+  SavedVariable(const SavedVariable&) = delete;
```

Comment:
Are these autogenerated. What's the logic behidn disabling copy ctor?

Reply:
Not generated. It's reasonable to delete them because variables may have some kind of reference counts.

Reply:
These copy ctor are already disabled today anyways due to the way the class is defined right? This is just making it explicit.

Reply:
@albanD They may be defined, but they are not used, otherwise CI would break.

Reply:
@zou3519 was actually complaining to me that they don't exist haha